app/beta/app.go

```diff
@@ -536,8 +536,8 @@ func New(
 		ibctransfertypes.ModuleName,
 		authz.ModuleName,
 		feegrant.ModuleName,
-		leveragetypes.ModuleName,
 		oracletypes.ModuleName,
+		leveragetypes.ModuleName,
```

Comment:
Just curious, what determines the init order for `oracle` and `leverage`?

I know oracle's allow list depends on leverage's allow list, but don't know if it's required during genesis, for example.

Reply:
Yeah so the reason oracle comes before leverage, is because there's a side effect of where x/leverage calls a hook, which x/oracle implements and so x/oracle's state needs to be set first.